modules/jupyter/outputs.tf

```diff
@@ -12,12 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-output "jupyterhub_uri" {
-  value = var.add_auth ? module.iap_auth[0].domain : (data.kubernetes_service.jupyter-ingress.status != null ? (data.kubernetes_service.jupyter-ingress.status[0].load_balancer != null ? "${data.kubernetes_service.jupyter-ingress.status[0].load_balancer[0].ingress[0].ip}" : "") : "")
+output "jupyter_domain" {
+  value = var.add_auth ? module.iap_auth[0].jupyter_domain : var.url_domain_addr
 }
 
-output "jupyterhub_password" {
+output "jupyter_password" {
```

Comment:
can we keep the same name for outputs.. they are refered at multiple other places as well